feat: additional ServiceAccount permissions and roles

declanwilson-lw opened the pull request:
Allow Org level and Project level integration ServiceAccount, additional roles
 and permissions to support Lacework Platform upcoming release with new report
 type, for Compliance analysis within GCP: CIS Security 1.2 Benchmark.
 https://workbench.cisecurity.org/benchmarks/5205

Signed-off-by: Declan Wilson <declan.wilson@lacework.net>

declanwilson-lw committed:
feat: additional ServiceAccount permissions and roles

Allow Org level and Project level integration ServiceAccount, additional roles
 and permissions to support Lacework Platform upcoming release with new report
 type, for Compliance analysis within GCP: CIS Security 1.2 Benchmark.
 https://workbench.cisecurity.org/benchmarks/5205

Signed-off-by: Declan Wilson <declan.wilson@lacework.net>

afiune commented on main.tf:
```diff
@@ -14,11 +14,13 @@ locals {
     "roles/browser",
-    "roles/iam.securityReviewer"
+    "roles/iam.securityReviewer",
+    "roles/iam.serviceAccountTokenCreator
```
```suggestion
    "roles/iam.serviceAccountTokenCreator"
```

declanwilson-lw replied:
fixed

afiune commented on main.tf:
```diff
@@ -14,11 +14,13 @@ locals {
   default_organization_roles = [
     "roles/browser",
     "roles/iam.securityReviewer",
+    "roles/iam.serviceAccountTokenCreator",
```
```suggestion
    "roles/iam.serviceAccountTokenCreator"
```

declanwilson-lw replied:
fixed

afiune reviewed (changes requested):
The pipeline pointed this out! 😅

declanwilson-lw committed:
feat: additional ServiceAccount permissions and roles

Allow Org level and Project level integration ServiceAccount, additional roles
 and permissions to support Lacework Platform upcoming release with new report
 type, for Compliance analysis within GCP: CIS Security 1.2 Benchmark.
 https://workbench.cisecurity.org/benchmarks/5205

Signed-off-by: Declan Wilson <declan.wilson@lacework.net>

afiune reviewed (approved):
![tenor-43838116](https://user-images.githubusercontent.com/5712253/132375998-1450654e-af27-45d8-9eb8-33f14e6158d3.gif)
